archai/discrete_search/search_spaces/nlp/tfpp/mixed_attention.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional
+
+import torch
+from torch import nn
+from transformers.pytorch_utils import Conv1D
+from transformers.models.gpt2.configuration_gpt2 import GPT2Config
+
+from archai.discrete_search.search_spaces.config import ArchConfig
+from archai.discrete_search.search_spaces.nlp.tfpp.ops import OPS
+
+
+class MixedAttentionBlock(nn.Module):
+    def __init__(self, arch_config: ArchConfig, hf_config: GPT2Config,
```

Comment:
TF++ is hard-coded to use CodeGen, right? If so, I think we should either use `CodeGenConfig` or `PretrainedConfig` for the `hf_config` type annotation.